### PR TITLE
Tamara bv665/3 multiplication

### PR DESCRIPTION
**Pull request by TamaraBV665**

### CODE REVIEW

**Review by TamaraBV665 — COMMENTED** [review]
Changed.

File: pythoncalcultor/multiply.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```
Comment (KAssersohn) [inline pythoncalcultor/multiply.py:2]: Looks like a typo here, I have suggested a change here
```suggestion
    return x * y
```

**Review by KAssersohn — CHANGES_REQUESTED** [review]
I've just made one suggested change

**Review by TamaraBV665 — COMMENTED** [review]
Change made.

File: pythoncalculator/__init__.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
-from .add import add 
+from .multiply import multiply
```
Comment (KAssersohn) [inline pythoncalculator/__init__.py:2]: Just noticed that the first line shouldn't have been deleted, try this instead 
```suggestion
from .add import add
from .multiply import multiply
```

**Review by KAssersohn — CHANGES_REQUESTED** [review]
Just noticed a missing line in this bit, suggested the correct code

File: pythoncalculator/__init__.py [hunk 3]
```diff
@@ -1,3 +1,3 @@
 from .add import add 
 from .subtract import subtract
```
Comment (KAssersohn) [inline pythoncalculator/__init__.py:2]:
```suggestion
```

**Review by KAssersohn — COMMENTED** [review]
Okay, maybe try this?

File: pythoncalcultor/multiply.py
Comment (ns-rse) [inline pythoncalcultor/multiply.py:1]: Hi @TamaraBV665 this is in the wrong directory, its needs to be in `pythoncalculator` directory.

**Review by ns-rse — CHANGES_REQUESTED** [review]
Hi @TamaraBV665 I spotted the `mulultiply.py` file was created in the wrong directory which means it can't be imported and the tests fail.  If you are able to move the file to the correct directory and push the changes everything should "Just Work(TM)" :grin:

**Review by KAssersohn — COMMENTED** [review]
This looks good i think ready to commit?

**Pull request closed without merging**